tests/cypress/tests/server/loading_uneditable_test.ts

```diff
@@ -22,34 +22,56 @@ describe('loading states uneditable', () => {
         // Table is not editable
         DashTable
             .getCell(0, 0, State.Loading)
-            .click()
-            .find('.dash-input-cell-value-container > input').should('have.length', 0);
-
-        DOM.focused.type(`Hello${Key.Enter}`);
-        DashTable.getCell(1, 0, State.Any).click();
+            .click();
 
+        DOM.focused.type(`Hello`);
         DashTable
             .getCell(0, 0, State.Any)
             .within(() => cy.get('.dash-cell-value')
-                .should('not.have.html', 'Hello'));
+                .should('not.have.value', 'Hello'));
 
         cy.wait(5000);
 
         // Table is editable
+        DashTable.getCell(0, 0, State.Ready);
+
+        DOM.focused.type(`Hello`);
         DashTable
             .getCell(0, 0)
-            .click()
-            .find('.dash-input-cell-value-container > input').should('have.length', 1);
+            .within(() => cy.get('.dash-cell-value')
+                .should('have.value', 'Hello'));
+    });
```

Comment:
Made the test a bit tighter by skipping Enter / Click, gives the table less chance to render multiple times and hide issues for renders with a smaller footprint (less props changed)

Reply:
I'm not excited about the `cy.wait(5000)` statements that are proliferating here... I won't block on this, given all the cypress headaches already in this PR, but it would be great if at some point we could figure out a condition to wait for rather than a time.

Reply:
Reverting. I ran this in `3.4.1` but in `3.6.1` I'm encountering weird timing issues.

Reply:
About the `cy.wait(5000)` the callback waits for 5 seconds and the maximum default timeout for a selector to succeed in Cypress is 4000ms. It is a bit yucky.